rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityFilter.vue

```diff
@@ -216,9 +218,14 @@
       </div>
       <template #footer>
         <btn @click="filterOpen = false">{{ $t("cancel") }}</btn>
-        <btn type="primary" class="btn btn-primary" @click="search">{{
-          $t("search")
-        }}</btn>
+        <btn
+          ref="searchButton"
```

Comment:
please remove the refs that aren't being used, it's preferable to use data-testids for tests whenever it's possible